.github/workflows/repo_file_for_test.repos

```diff
@@ -7,12 +7,12 @@ repositories:
   ament_cmake:
     type: git
     url: https://github.com/ament/ament_cmake.git
-    version: 0.8.1
+    version: 1.3.11
```

Comment:
The tags here can't be too recent (this one is from 6 months ago), since this file is used for Humble tests, for example here: https://github.com/ros-tooling/action-ros-ci/blob/8dce0379055992c29df0f6fe13d5d47b75b70aec/.github/workflows/test.yml#L315.

You could update to the last version/tag for Humble.

Reply:
I have inserted the tags to support the oldest supported ros distro, humble.

you can see that the most recent version of ament_cmake is 2.8.1 ..I used the v. 1.3.11 .. by using version : humble 

if you use the tag with the name of the distro you want to test, it is also easier to maintain the action-ros-ci repository